src/main/java/org/alliancegenome/curation_api/jobs/executors/GeneExpressionExecutor.java

```diff
@@ -39,21 +46,47 @@ public void execLoad(BulkLoadFileHistory bulkLoadFileHistory) {
 
 			bulkLoadFileHistory.setCount(geneExpressionIngestFmsDTO.getData().size());
 			updateHistory(bulkLoadFileHistory);
-			
+
 			List<Long> annotationIdsLoaded = new ArrayList<>();
 			List<Long> annotationIdsBefore = geneExpressionAnnotationService.getAnnotationIdsByDataProvider(dataProvider);
 
 			boolean success = runLoad(geneExpressionAnnotationService, bulkLoadFileHistory, dataProvider, geneExpressionIngestFmsDTO.getData(), annotationIdsLoaded);
+
 			if (success) {
 				runCleanup(geneExpressionAnnotationService, bulkLoadFileHistory, dataProvider.name(), annotationIdsBefore, annotationIdsLoaded, "gene expression annotation");
+				loadExperiments(bulkLoadFileHistory);
```

Comment:
Presumably there needs to be some sort of mechanism to clean up experiments too?

Reply:
I don't know a simple way to define a rule to obsolete an experiment (defined by a triple paperId, assayId, geneId)

Reply:
Same as any other load I would have thought.  Check experiment IDs before load, record IDs of experiments created/updated, compare lists at end.

Reply:
Or maybe if an experiment has (eventually) no annotations linked. But maybe they could be kept for historical reasons?

Reply:
I would have thought that you would want to deprecate (make obsolete) an experiment if all the annotations relating to that experiment had been deprecated.

Reply:
cleanup added